scripts/linters/pylint_extensions.py

```diff
@@ -337,7 +337,12 @@ def process_tokens(self, tokens):
                         if comment_index == 0:
                             continue
                         else:
-                            if split_content[comment_index - 1].endswith('('):
+                            last_content_before_comment = (
+                                split_content[comment_index - 1])
+                            if (
+                                    last_content_before_comment.endswith('(') or
+                                    last_content_before_comment.endswith('[')
+                            ):
```

Comment:
You can pass a tuple of strings to `endswith`:
```suggestion
                            if last_content_before_comment.endswith(('(', '[')):
```

Reply:
Ah, right, thanks!